### ISSUE TITLE
Tasten Backspace und Entfernen verhalten sich nicht wie erwartet

### ISSUE THREAD

**Issue by KarlZeilhofer** [ElektronikNode/scicalc#1]
* Gesamten Text mit Strg+A markieren
* Entfernen oder Backspace drücken
* Inhalt des Texteditors bleibt unverändert
* Erwartet: Es sollte alles gelöscht werden

**Comment by feichtinger** [comment]
Ja, dieser (und einige weitere Bugs) hängen damit zusammen, dass ich Ein- und Ausgabe im selben Textfenster haben wollte. Dazu habe ich viel herum tricksen müssen, was zu einigen merkwürdigen Effekten führte.

**Comment by KarlZeilhofer** [comment]
wurde in cddf52b8b4f7fd08c4a6e290864ef401aacb1564 behoben.
